superset/assets/stylesheets/superset.less

```diff
@@ -139,6 +139,14 @@ span.title-block {
 }
 div.navbar {
   z-index: 999;
+  .dropdown-menu .fineprint{
+    padding: 10px 20px 0 20px;
```

Comment:
Can be `padding: 10px 20px 0;`

Reply:
I typically prefer the verbose version since it's faster for people to visualize when reading, but if there is a standard/preference, I'm open to it.

Reply:
Seconded on preferring the verbose version. Three-parameter padding/margin always confuse me and I have to look it up.